x-pack/plugins/security_solution/public/cloud_security_posture/components/entity_insight.tsx

```diff
@@ -103,14 +108,23 @@ export const EntityInsight = <T,>({
   if (hasMisconfigurationFindings)
     insightContent.push(
       <>
-        <MisconfigurationsPreview name={name} fieldName={fieldName} isPreviewMode={isPreviewMode} />
+        <MisconfigurationsPreview
+          name={name}
+          fieldName={fieldName}
+          hasNonClosedAlerts={alertsCount > 0}
```

Comment:
It feels conceptually incorrect that the Misconfiguration and Vulnerability preview should know about non-closed alerts. Why do we need it here?

Reply:
same reason as the checking existence of Misconfiguration and Vulnerability in Alerts Preview, we need to pass this information when opening left panel so it knows what tabs gets rendered